src/lib/memory/homeobject.hpp

```diff
@@ -32,11 +32,18 @@ struct std::hash< homeobject::BlobRoute > {
 
 namespace homeobject {
 
+using btree = std::unordered_map< BlobRoute, Blob >;
+
+struct ShardIndex {
+    mutable std::shared_mutex _btree_lock;
+    btree _btree;
```

Comment:
`bool sealed` ?

Reply:
sealed is stored in the metadata (ShardInfo) in the PGMap